common/lib/capa/capa/customrender.py

```diff
@@ -137,3 +136,33 @@ def get_html(self):
         return xhtml
 
 registry.register(TargetedFeedbackRenderer)
+
+#-----------------------------------------------------------------------------
+
+
+class ClarificationRenderer(object):
+    """
+    A clarification appears as an inline [?] which reveals more information when the user
+    hovers over it.
+
+    e.g. <p>Enter the ROA <clarification>Return on Assets</clarification> for 2015:</p>
+    """
+    tags = ['clarification']
+
+    def __init__(self, system, xml):
+        self.system = system
+        initial_text = xml.text if xml.text else ''  # Necessary since xml.text can be None type
```

Comment:
Nit: style wise, prefer comments that explain a line of code to appear the line above the line in question. Enhances readability.

Same nit at L165